.eslintrc.cjs

```diff
@@ -1,48 +1,12 @@
 module.exports = {
-  extends: [
-    '@mediamonks/eslint-config-base',
-    'plugin:lit/recommended',
-    'plugin:lit-a11y/recommended',
-  ],
   parserOptions: {
     extraFileExtensions: ['.cjs'],
     project: './tsconfig.json',
   },
+  extends: [
+    '@muban/eslint-config',
+  ],
   rules: {
-    // Additions
-    '@typescript-eslint/consistent-type-imports': ['error'],
-    '@typescript-eslint/naming-convention': [
-      'error',
-      {
-        selector: 'default',
-        format: ['strictCamelCase'],
-        leadingUnderscore: 'forbid',
-        trailingUnderscore: 'forbid',
-      },
-      {
-        selector: 'typeLike',
-        format: ['StrictPascalCase'],
-      },
-      {
-        selector: 'variable',
-        // Exception for FunctionComponents
-        format: ['strictCamelCase', 'StrictPascalCase', 'UPPER_CASE'],
-      },
-      {
-        selector: 'function',
-        // Exception for FunctionComponents
-        format: ['strictCamelCase', 'StrictPascalCase'],
-      },
-      {
-        selector: 'enumMember',
-        format: ['StrictPascalCase'],
-      },
-    ],
-    'lit/no-legacy-template-syntax': 'off',
-    'lit/no-private-properties': 'off',
-    'lit/no-property-change-update': 'off',
-    'lit/no-template-map': 'off',
-    'lit/binding-positions': 'off',
-    'lit/no-invalid-html': 'off',
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.ts"]}]
```

Comment:
the skeleton doesn't have any testing framework setup, so this doesn't add anything. However, we should allow `devDependencies` in `'./mocks/**/*.ts'` and `'**/*.stories.ts'`, as those files are part of the development environment.

Reply:
I still like to keep `test` in there for when people to add tests themselves, but I'll add the others.

> the skeleton doesn't have any testing framework setup
Probably something we should add soon :D